src/test/java/EditCommandTest.java

```diff
@@ -0,0 +1,8 @@
+import duke.command.EditCommand;
+import org.junit.jupiter.api.Test;
+
+public class EditCommandTest {
+
+   // @Test
+
```

Comment:
I would suggest that you guys write some test code, as and when you update the code